### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by rgeraldporter**

### CODE REVIEW

**Review by CharleneRadke — APPROVED** [review]

**Review by JenChampagne — APPROVED** [review]

**Review by AdamSutch — APPROVED** [review]

**Review by AdamSutch — CHANGES_REQUESTED** [review]
Umm hold up, this is a public repo. Do we really want this template in that?

**Comment by rgeraldporter** [comment]
> Umm hold up, this is a public repo. Do we really want this template in that?

Ah, true. Do we have a way to make an exception in Vanta for this?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+### JIRA Ticket Link
```
Comment (williambuttenham) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion

```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+*In your own words, what does this code change accomplish?*
+
+
+
+### Interoperability & Impact Notes
+*Include notes in the space below about what systems and products -- internal or external to this repository -- that reviewers should consider when assessing the code changes? If you aren't sure, please ask in Slack for suggestions.*
+
+
+
+### Other considerations
```
Comment (williambuttenham) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+- [ ] Hours are logged in JIRA ticket (always required)
```
Comment (williambuttenham) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
```

**Comment by williambuttenham** [comment]
I think this could still work if we remove these fields?

**Review by williambuttenham — APPROVED** [review]

**Review by williambuttenham — APPROVED** [review]
Looks good!

**Pull request merged**